### PR TITLE
feat(route): add 簡訊設計

### LINKED ISSUE #8173: 志祺七七
### 网站地址
https://blog.simpleinfo.cc/shasha77

### 网站描述
https://www.youtube.com/channel/UCiWXd0nmBjlKROwzMyPV-Nw

### 需要生成什么内容？
https://blog.simpleinfo.cc/shasha77

### 额外描述
image

### PR DESCRIPTION
**Pull request by haukeng**
## 该 PR 相关 Issue / Involved issue

Close #8173

## 完整路由地址 / Example for the proposed route(s)

```routes
/simpleinfo
/simpleinfo/talk
/simpleinfo/work
/simpleinfo/news
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [x] 全文获取 fulltext
  - [x] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [x] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/simpleinfo/index.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const timezone = require('@/utils/timezone');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const { category } = ctx.params;
+    const rootUrl = 'https://blog.simpleinfo.cc';
+    const link = `${rootUrl}${category ? (category === 'work' || category === 'talk' ? `/blog/${category}` : `/shasha77?category=${category}`) : '/shasha77'}`;
+    const response = await got.get(link);
```
Comment (TonyRL) [inline lib/v2/simpleinfo/index.js:10]:
```suggestion
    const response = await got(link);
```

File: lib/v2/simpleinfo/index.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        list.map(async (item) => {
+            await ctx.cache.tryGet(item.link, async () => {
+                const result = await got.get(item.link);
+                const content = cheerio.load(result.data);
+                item.author = content('meta[property="article:author"]').attr('content');
+                item.pubDate = timezone(parseDate(content('meta[property="article:published_time"]').attr('content')), +8);
+                item.description = `<img src="${content('meta[property="og:image"]').attr('content')}">${content('.article-content').first().html()}`;
+            });
+            return item;
+        })
```
Comment (TonyRL) [inline lib/v2/simpleinfo/index.js:36]:
```suggestion
        list.map((item) =>
            ctx.cache.tryGet(item.link, async () => {
                const result = await got(item.link);
                const content = cheerio.load(result.data);
                item.author = content('meta[property="article:author"]').attr('content');
                item.pubDate = timezone(parseDate(content('meta[property="article:published_time"]').attr('content')), +8);
                item.description = `<img src="${content('meta[property="og:image"]').attr('content')}">${content('.article-content').first().html()}`;
                return item;
            })
        )
```

File: lib/v2/simpleinfo/index.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        .get();
+
+    const items = await Promise.all(
+        list.map(async (item) => {
+            await ctx.cache.tryGet(item.link, async () => {
+                const result = await got.get(item.link);
+                const content = cheerio.load(result.data);
+                item.author = content('meta[property="article:author"]').attr('content');
+                item.pubDate = timezone(parseDate(content('meta[property="article:published_time"]').attr('content')), +8);
+                item.description = `<img src="${content('meta[property="og:image"]').attr('content')}">${content('.article-content').first().html()}`;
```
Comment (TonyRL) [inline lib/v2/simpleinfo/index.js:33]: Please use `art-template`
docs: https://docs.rsshub.app/joinus/script-standard.html#template

Comment (haukeng) [reply lib/v2/simpleinfo/index.js:33]: okay, I will change this.

**Pull request merged**